README.md

```diff
@@ -127,6 +144,82 @@ OutlineView(data, children: \.children, selection: $selection) { item in
 .rowSeparatorColor(.red)
 ```
 
+### Drag & Drop
+
+#### Dragging From `OutlineView`
+
+Add the `dragDataSource` modifier to the `OutlineView` to allow dragging rows from the `OutlineView`. The `dragDataSource` takes a closure that translates a data element into an optional `NSPasteboardItem` (with a nil value meaning the row can't be dragged).
+
+```swift
+extension NSPasteboard.PasteboardType {
+  static var myPasteboardType: Self {
+    PasteboardType("MySpecialPasteboardIdentifier")
+  }
+}
+
+theOutlineView
+  .dragDataSource { item in
+    let pbItem = NSPasteboardItem()
+    pbItem.setData(item.dataRepresentation, forType: .myPasteboardType)
+    return pbItem
+  }
+```
+
+#### Dropping into `OutlineView`
+
+Handling drag events that come into the `OutlineView`, either from `dragDataSource` modifier or from outside the `OutlineView`, can be handled by adding the modifier `onDrop(of:receiver:)` This modifier gives the `OutlineView` a list of `NSPasteboard.PasteboardType` that can be read, and supplies a delegate object, conforming to the `DropReceiver` protocol. `DropReceiver` implements functions to validate a drop operation, read items from the dragging pasteboard, and update the data source when a drop is successful.
+
+```swift
+theOutlineView
```

Comment:
```suggestion
outlineView
```

Reply:
Got it! https://github.com/Sameesunkaria/OutlineView/pull/10/commits/9a66eb8d6576eb16ca2d046bf17d013ac32249e3